WordPress/Classes/ViewRelated/Gutenberg/EditorMediaUtility.swift

```diff
@@ -8,66 +8,24 @@ final class AuthenticatedImageDownload: AsyncOperation {
     }
 
     let url: URL
-    let blogObjectID: NSManagedObjectID
+    let mediaHost: MediaHost
```

Comment:
From the PR description:

> `AuthenticatedImageDownload` no long access `NSManagedObject`. It used to have code to query `Blog`. The code is now moved to its caller.

Just wanted to +1 this decision. Aside from the hunch that this will help with #20630, having the image download operation know about `MediaHost` over `Blog` seems like a better design because the surface of information it can access (thus that we need to be aware of when working with it) is now much smaller.